plugins/services/src/js/reducers/serviceForm/Container.js

```diff
@@ -151,7 +151,7 @@ const containerJSONReducer = combineReducers({
 
     // Convert portDefinitions to portMappings
     return this.portDefinitions.map((portDefinition, index) => {
-      const vipLabel = `VIP_${index}`;
+      const vipLabel = portDefinition.vipLabel || `VIP_${index}`;
```

Comment:
Probably not super important, but the default label shows up a few places `VIP_${index}`, worth making that a constant/helper somewhere?

Reply:
It was a bug fix PR, I added that refactoring, please test thoroughly.